src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -116,35 +186,91 @@
         <div class="modal-content">
             <h4>Theme</h4>
 
-            <input class="config-option" type="radio" name="theme" value="auto" id="settings-theme-auto">
-            <label for="settings-theme-auto">Auto</label>
-            <p class="form-help"><strong>Default theme</strong>. It switches between Light and Dark automatically to match the operating system theme.</p>
+            <div class="settings-group">
+                <label for="settings-theme-auto">
+                    <input class="config-option" type="radio" name="theme" value="auto" id="settings-theme-auto">
+                    <p>
+                        <svg xmlns="http://www.w3.org/2000/svg" data-icon-name="icon-tabler-device-laptop" width="24" height="24" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
```

Comment:
can we have those icons as svg files instead, as done for other icons ? This way, github shows a preview for them too for instance.